drivers/mipi_dsi/dsi_mcux_2l.c

```diff
@@ -200,35 +208,39 @@ static int mcux_mipi_dsi_init(const struct device *dev)
 	return 0;
 }
 
+#define MCUX_DSI_DPI_CONFIG(id)									\
+	IF_ENABLED(DT_NODE_HAS_PROP(DT_DRV_INST(id), nxp_lcdif),				\
+	(.dpi_config = {									\
+		.dpiColorCoding = DT_INST_ENUM_IDX(id, dpi_color_coding),			\
```

Comment:
How about having the macro at this level? Like
```c

#define MCUX_DSI_DPI_CONFIG_INIT(id) { \
        ..dpiColorCoding = ...
```

That's what I had in mind when I suggested to bring it out (sorry I was unclear), thinking that doing that would get rid of another tab and then you'd actually be able to keep more stuff inline.

The kernel structure have these all over, example: https://github.com/zephyrproject-rtos/zephyr/blob/main/include/zephyr/kernel.h#L695-L710

Reply:
Sorry I misunderstood- thanks for clarifying. Should be fixed to this format in latest push

Reply:
Thanks, I feel a bit bad that after making you redo this twice most lines still don't fit without wrapping by just one character but it's probably good to have it split out anyway :-)

Reply:
No worries- still looks cleaner overall